src/extractors/custom/www.newyorker.com/index.js

```diff
@@ -4,15 +4,15 @@
 export const NewYorkerExtractor = {
   domain: 'www.newyorker.com',
   title: {
-    selectors: ['h1.title'],
+    selectors: ['h1[class^="ArticleHeader__hed"]'],
   },
 
   author: {
-    selectors: ['.contributors'],
+    selectors: ['div[class^="ArticleContributors"] a[rel="author"]'],
```

Comment:
In some cases, articles could have multiple authors (for example https://www.newyorker.com/humor/daily-shouts/teas-you-should-probably-get-rid-of-already), would it be possible to use a selectors that handles both cases, or if needed, two selectors (the second being a fallback for when this one doesn't return 1 match)?